test_project/tests/integration_tests/test_admin/test_export.py

```diff
@@ -49,3 +52,123 @@ def test_export_using_admin_model(client: Client, superuser: User):
     assert ExportJob.objects.exists()
     export_job = ExportJob.objects.first()
     assert export_job.export_status == ExportJob.ExportStatus.EXPORTED
+
+
+@pytest.mark.django_db(transaction=True)
+def test_export_progress_for_sync_mode(
```

Comment:
Is there a difference between sync/async modes now?

Please, provide more detailed docstring (why there are two different tests for sync and async and how it works)